pkg/build/registry/buildlog/rest.go

```diff
@@ -84,11 +84,11 @@ func (r *REST) Get(ctx kapi.Context, name string, opts runtime.Object) (runtime.
 
 	// The build was cancelled
 	case api.BuildStatusCancelled:
-		return nil, fmt.Errorf("Build %s/%s was cancelled", build.Namespace, build.Name)
+		return nil, errors.NewBadRequest(fmt.Sprintf("Build %s/%s was cancelled", build.Namespace, build.Name))
```

Comment:
lower case - `Build` should be `build` since this will be written like "error: build a/b was cancelled"